xml/System/HashCode.xml

```diff
@@ -13,8 +13,54 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Combines the hash code for multiple values into a single hash code.</summary>
+    <remarks>
+      <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+You can use <xref:System.HashCode> to combine multiple values (for example, fields on a structure or class) into a
+single hash code using the xxHash32 algorithm. This structure operates in one of two ways:
+
+ * Static methods that accept a set of up to eight values to combine.
+ * Two instance methods that operate in a streaming fashion, accepting value one at a time.
+
+### Static Methods
+
+The static methods should be preferred when eight or fewer hash codes are being combined.
+ 
+ [!code-csharp[System.HashCode.Structure#1](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.hashcode.structure/cs/example1.cs#1)]
```

Comment:
Where do I add these samples?

Reply:
See [the sample repository](https://github.com/dotnet/samples)

Reply:
https://github.com/dotnet/samples/pull/92